code/modules/mob/living/simple_mob/subtypes/vore/rat.dm

```diff
@@ -45,7 +45,7 @@
 	say_list_type = /datum/say_list/ratte
 	ai_holder_type = /datum/ai_holder/simple_mob/melee/rat
 
-/mob/living/simple_mob/vore/rat/passive
+/mob/living/simple_mob/vore/aggresive/rat/passive		//Yes it makes no sense. But for now it will work
```

Comment:
Mob being aggressive and passive? I never fixed the foodcode so it's either or not both currently.

Reply:
better wording implemented.